staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -128,8 +61,12 @@ func NewDeltaFIFOWithOptions(opts DeltaFIFOOptions) *DeltaFIFO {
 // Deleted if the older Deleted's object is a
 // DeletedFinalStateUnknown.
 //
-// The other difference is that DeltaFIFO has an additional way that
-// an object can be applied to an accumulator, called Sync.
+// The second difference is that DeltaFIFO has an additional way that
+// an object can be applied to an accumulator, called Replaced.  However,
+// if EmitDeltaTypeReplaced is not set to true, Sync will be used in
+// replace events for backwards compatibility.
+//
+// The last difference is that Sync is used for periodic resync events.
```

Comment:
"Sync" is _introduced_ by DeltaFIFO, just as "Replaced" is.  I would organize things this way:

DeltaFIFO is like FIFO but differs in two ways.
...
The other difference is that DeltaFIFO has two additional ways that an object can be applied to an accumulator: Replaced and Sync. ...